mgr/src/cmds/rebalance_process.cr

```diff
@@ -247,6 +248,16 @@ class Rebalancer
     # TODO: Handle Mount failure and other errors
     execute("glusterfs", args)
   end
+
+  def umount
+    # TODO: Handle umount errors except mount not exists
+    execute("umount", [@mount_dir])
+
+    # TODO: Handle errors
+    execute("chattr", ["-i", @mount_dir])
+
+    FileUtils.rm_rf(@mount_dir)
```

Comment:
Use `rmdir` for the safer side. If mount directory is already unmounted and some other files exists, `rm -rf` may be harmful.

Reply:
Ack! Done